staging/src/k8s.io/pod-security-admission/policy/check_capabilities_baseline.go

```diff
@@ -80,6 +80,14 @@ func capabilitiesBaseline_1_0(podMetadata *metav1.ObjectMeta, podSpec *corev1.Po
 	var badContainers []string
 	nonDefaultCapabilities := sets.NewString()
 	visitContainers(podSpec, func(container *corev1.Container) {
+		// Skip validation of capabilities if the pod is in a user namespace
+		// (and the "UserNamespacesPodSecurityStandards" feature gate is enabled).
+		// The baseline policy does not enforce a seccomp profile, which is the only way
+		// to prevent a pod from using the `unshare` syscall. Thus, a user in a baseline namespace
```

Comment:
> The baseline policy does not enforce a seccomp profile, which is the only way to prevent a pod from using the `unshare` syscall.

I still don't think this is accurate when the default seccomp feature is in use. Baseline prevents pods from opting into unconfined, default seccomp makes those pods have a seccomp profile be applied.

What is the plan to reconcile this API-level relaxation with node-level default seccomp configuration? Unless that is done, I don't think this can progress to be enabled by default. We need a plan to how this eventually progresses to be on by default in a way that doesn't break default seccomp before merge.